climateconnect/climateconnect
# Feature: Allow adding custom image to hub footer

## What and Why
After making a grant to fund the start of the ClimateHub the city of Potsdam has requested that we show their "Funded by the city of Potsdam" logo on the page. Therefore I added the generic option to add a custom footer image to a hub which will then be shown instead of our current "Made with :heart: for :earth_africa:"

![Screenshot from 2023-02-27 08-02-09](https://user-images.githubusercontent.com/10531079/221496654-2cd57ecc-5738-4d32-afc0-20e12b6061c8.png)

## Concrete changes
- Added a new property `custom_footer_image` to the `hub` model 
- Added code to pass the image through to the footer and display it instead of "Made with :heart: for :earth_africa:"

## Testing guide
1. Run migrations
2. Create a hub on your local server
3. Add a custom footer image to the hub
4. Visit the hub and scroll all the way to the bottom
5. See the image
